config/intellij-idea-inspections.xml

```diff
@@ -1997,6 +1997,29 @@
     <inspection_tool class="SuperClassHasFrequentlyUsedInheritors" enabled="true" level="ERROR" enabled_by_default="true" />
     <inspection_tool class="SuperTearDownInFinally" enabled="false" level="ERROR" enabled_by_default="false" />
     <inspection_tool class="SuppressionAnnotation" enabled="true" level="ERROR" enabled_by_default="true">
+        <scope name="Tests" level="ERROR" enabled="true">
```

Comment:
my previous tries were not successful so I tried to create another scope

this is my previous configuration
```
<inspection_tool class="SuppressionAnnotation" enabled="true" level="ERROR" enabled_by_default="true">
        <option name="myAllowedSuppressions">
            <list>
                <option value="deprecation" />
                <option value="unchecked" />
                <option value="rawtypes" />
                <!-- There is no other way to deliver filename that was under processing.
                     See https://github.com/checkstyle/checkstyle/issues/2285-->
                <option value="ProhibitedExceptionThrown" />
                <option value="MismatchedQueryAndUpdateOfCollection" />
                <!-- Till https://github.com/checkstyle/checkstyle/issues/3066 -->
                <option value="ProhibitedExceptionCaught" />
                <!-- No way to split apart huge if/else branches in test. -->
                <option value="IfStatementWithTooManyBranches" />
                <!-- we have to catch Exception in Checker and rethrow it -->
                <option value="ProhibitedExceptionThrown" />
                <!-- we have to use it when pass null argument in test purporses -->
                <option value="NullArgumentToVariableArgMethod" />
 <!-- we have to use it because number of dependencies of CheckerTest exceeds 33. -->
                    <option value="ClassWithTooManyDependencies" />
            </list>
        </option>
    </inspection_tool>
```